examples/includes-in-sourceset/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+import software.amazon.smithy.gradle.tasks.SmithyBuildTask
+
+// This example builds Smithy models using a custom build task
```

Comment:
Are these going to stay here? Or move all the docs to README.md?

Reply:
I think it makes sense to move it all into the README. I will do that as part of the docs pass